hooks/useInfiniteBlocks.tsx

```diff
@@ -24,7 +26,10 @@ const useInfiniteBlocks = (
         .then(data =>
           setBlocksData(prevData => ({
             ...data,
-            data: prevData.data.concat(data.data),
+            data: sort(
+              (blockA: BlockType, blockB: BlockType) => blockB.sequence - blockA.sequence, 
+              uniqBy(block => block.id, prevData.data.concat(data.data))
```

Comment:
Trivial but you can also use `prop('id')` or `propOr(-1, 'id')` or the `safeProp` function I put in `utils`